tests/unit/Certify.spec.ts

```diff
@@ -62,6 +63,7 @@ function createComponent (
   store.state.currentDate = currentDate
 
   return mount(Certify, {
+    vuetify,
```

Comment:
This caused all manners of problems with updated imports. I have no idea why this worked before (since the tests expect Vuetify components to be rendered not stubbed).